tests/output/sql/output_test.js

```diff
@@ -116,6 +124,13 @@ describe("Linting", function() {
         "INSERT INTO character_episodes VALUES (2, 'Rick');" +
         "SELECT * FROM characters LEFT OUTER JOIN character_episodes ON " +
         "id = characterId");
+    failingTest("Ambiguous column name when joining",
+	"CREATE TABLE students (id INTEGER, name TEXT);" +
+	"CREATE TABLE student_grades (id INTEGER, grade INTEGER);" +
+	"INSERT INTO students VALUES (1, \"Brian\");" +
+	"INSERT INTO student_grades VALUES (1, 95);" +
+	"SELECT id FROM students INNER JOIN student_grades on students.id = student_grades.id;",
+	["Ambiguous column name \"id\"."]);
```

Comment:
How does this test relate to the change?

Reply:
I did not know how to add tests before so I added an additional test covering previous PR #717.

Reply:
I moved the test to #729 since it would have to be updated anyways.